Taskfile.yml

```diff
@@ -93,7 +93,7 @@ tasks:
     desc: Build the Go code
     dir: "{{.DEFAULT_GO_MODULE_PATH}}"
     cmds:
-      - go build -v {{.LDFLAGS}}
+      - CGO_ENABLED=0 go build -v {{.LDFLAGS}}
```

Comment:
Are we sure that the serial operations are working correctly with CGO disabled? macOS I'm looking at you...

Reply:
Nope, we have to test it on mac. Unfortunately I don't have any at home :cry:

Reply:
@cmaglie could you please help with this?

Reply:
Tested on MacOSX Venutra 13.4.1 (Apple M1) and an Arduino Nano 33 IoT:
- ✅ Firmware Upload
- ✅ Firmware get-version
- ✅ Certificate upload